refinery_core/Cargo.toml

```
@@ -33,17 +33,14 @@ walkdir = "2.3.1"
 rusqlite = { version = ">= 0.23, <= 0.29", optional = true }
 postgres = { version = ">=0.17, <= 0.19", optional = true }
 tokio-postgres = { version = ">= 0.5, <= 0.7", optional = true }
-mysql = { version = ">= 21.0.0, <= 23", optional = true, default-features = false }
-mysql_async = { version = ">= 0.28, <= 0.32", optional = true }
+mysql = { version = ">= 21.0.0, <= 23", optional = true, default-features = false, features = ["minimal"] }
+mysql_async = { version = ">= 0.28, <= 0.32", optional = true, default-features = false, features = ["minimal"] }
 tiberius = { version = ">= 0.7, <= 0.12", optional = true, default-features = false }
-tokio = { version = "1.0", features = ["full"], optional = true }
-futures = { version = "0.3.16", optional = true }
-tokio-util = { version = "0.7.7", features = ["compat"], optional = true }
+tokio = { version = "1.0", default-features = true, optional = true }
```

```suggestion
tokio = { version = "1.0", optional = true }
```
default features is true by default